src/pages/tutorial/angular/step-2.mdx

```diff
@@ -0,0 +1,718 @@
+---
+title: 2. Building pages
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+import Preview from 'components/Preview';
+
+### Now that we have a base Angular app, it's time to build a few static pages. In this step, we'll become comfortable with the Carbon UI Shell, grid and various Carbon components.
+
+<AnchorLinks>
+
+<AnchorLink>Fork, clone and branch</AnchorLink>
+<AnchorLink>Add grid</AnchorLink>
+<AnchorLink>Add landing page grid</AnchorLink>
+<AnchorLink>Build landing page</AnchorLink>
+<AnchorLink>Style landing page</AnchorLink>
+<AnchorLink>Add repo page grid</AnchorLink>
+<AnchorLink>Build repo page</AnchorLink>
+<AnchorLink>Style repo page</AnchorLink>
+<AnchorLink>Submit pull request</AnchorLink>
+
+</AnchorLinks>
+
+## Preview
+
+A [preview](https://angular-step-3-carbon-tutorial.netlify.com) of what you'll build:
+
+<Preview
+  height="400"
+  title="Carbon Angular Tutorial Step 2"
+  src="https://angular-step-3-carbon-tutorial.netlify.com"
+  frameborder="no"
+  allowtransparency="true"
+  allowfullscreen={true}
+  class="bx--iframe bx--iframe--border"
+/>
+
+_Note: If you get lint errors when you copy the code from the snippets, run_ `ng lint --fix` _to fix them._
```

Comment:
Is this note specific to step 2 or should it be said in the overview or step 1?

Reply:
Added wherever it would be relevant :+1: (so, step 1, 2, 3, and 4)